test/credit_card_sanitizer_test.rb

```diff
@@ -102,6 +102,7 @@ class CreditCardSanitizerTest < MiniTest::Test
         assert_nil @sanitizer.sanitize!("blah blah  +4111111111111111.json")
         assert_nil @sanitizer.sanitize!("\"+4111111111111111\"")
         assert_nil @sanitizer.sanitize!("(+4111111111111111)")
+        assert_nil @sanitizer.sanitize!("&#43;1 936 321 1111")
```

Comment:
Actually, I just tested the following on the master branch before the fix and I can't get the bug to reproduce for any number except the one I was originally testing with:

```ruby
irb(main):008:0> CreditCardSanitizer.new.sanitize!("&#43;4111111111111111")
=> nil
irb(main):005:0> CreditCardSanitizer.new.sanitize!("&#43;1 936 321 1111")
=> "&#43;1 936 ▇▇▇ 1111"
```

Any thoughts as to why this number seems to be the only one being incorrectly redacted?